### ISSUE TITLE
Problem dra igång din klientapp

### ISSUE THREAD

**Issue by Grenmyr** [CCHolmgren/ch222kv_1dv450_kod#13]
Får detta felet när jag gårt till localhost:3000

Missing template static/index, application/index with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :coffee, :jbuilder]}. Searched in: \* "/home/dav/Hämtningar/ch222kv_1dv450_kod-master/RegistrationApplication/app/views" 

Jag hittar inte heller några instruktioner för installation, kröanvisningar eller något alls angående senaste release. Din angular app. Finns dem?

**Comment by Grenmyr** [comment]
Är du säker du är färdig med angular appen?  
För det finns exempelkod från johns föreläsningar, massa console.log() o grejjor i den katalogen public/assets du nämnde i denna issue https://github.com/ch222kv/ch222kv_1dv450_kod/issues/12 koden. Eller så har jag fått en gammal version av ditt repo.

Jag testade ladda ner din master branch applikation (dvs senaste release) också samma sak där, mycket exempelkod från john blandat med din.

**Comment by ch222kv** [comment]
Tiden tog helt enkelt slut. 
Dock ska funktionaliteten finnas där, bara att vissa saker ska tas bort, men det hann jag inte med. 
För att lösa templateproblemet så behöver det säkerligen skapas en index.html.erb fil i views/static/, konstig eftersom det fungerar för mig.

**Comment by Grenmyr** [comment]
Ja, tiden jag håller med, tiden för angular appen var väldigt kort. Jag har inte heller fått med allt jag ville. Jag tror det blev lite så för läsperioden är kortare i år.

Jag var jättestressad också, satt till 23:30 igår, så jag är rädd jag glömt få upp filer också, vi får se när jag blir peer-rewied vad jag missat.

Jag testar senare i helgen igen (kanske söndag) och försöker få igång din app. Lycka till i nästa läsperiod och grymt grupprojekt ni hade förrästen. Ni hade den där cube grejjen va? grym!

**Comment by 1dv450** [comment]
Jag får samma problem med ingången till applikationen. "Missing template static/index...". Kolla över så allt ligger på Github för den verkar saknas. Svårt för oss att få någon ingång till klientapplikationen annars.

**Comment by ch222kv** [comment]
Det är som sagt väldigt konstigt. Tidigaste jag kan kolla på det är ikväll, så då kan jag se över vad som är fel och få det att fungera.

**Comment by 1dv450** [comment]
:+1:

**Comment by ch222kv** [comment]
Problemet var att en filen hade lyckats smita undan att bli tillagd i gitrepot, men nu är den tillagd. Jag har också ändrat tillbaka till sqlite databasen, så slipper ni bråket med den. Detta gör dock att söknigen, och även fliken Search events inte fungerar.
